### ISSUE TITLE
Feature request: Web server 'busy' status

### ISSUE THREAD

**Issue by mmelendrez** [bioinformatics-centre/kaiju#118]
I've been attempting to upload files ~7.5GB in size. They seem to upload fine (it takes awhile) but when the progress bar hits 100% and I hit the submit button the page reloads saying there is an Error: no file uploaded, as if the server actually didn't upload the file even though it hit 100%. Given how long it is taking to upload these files, I am wondering if the server times out? Or some uploads seem to go faster than others as a consequence perhaps of server 'load'/'busy-ness'. Would be useful to know the server status to help me figure out if my file isn't uploading due to server load or something with the web submission form?

**Comment by pmenzel** [comment]
Hi, I just tested the web server and it went fine.
Maybe the file size was too big though, not sure what went wrong for you.
Also make sure to only use one submission form at once in your web browser and not multiple tabs/windows. This will cause problems.

**Comment by mmelendrez** [comment]
Yep - I think it was a combination of: (1) I was attempting to load two R1/R2 files - each 7.5GB so in effect 14GB+ was being uploaded, that could've been an issue on my end as it took overnight to get them fully loaded and the next morning the webpage had expired I think. (2) I did have another submission window open at the same time attempting to do a different file. (3) My upload speeds here at SCSU are on the slower side in my opinion (8-9 Mbps upload). End of day yesterday I got just my R1 up (7.5GB) and came in this morning and it was done with output. So the russian roulette of internet speed favored me last night. Thanks!
